### ISSUE TITLE
Problem when pod install

### ISSUE THREAD

**Issue by yarodevuci** [eBay/NMessenger#74]
The Podfile contains framework targets, for which the Podfile does not contain host targets (targets which embed the framework).
If this project is for doing framework development, you can ignore this message. Otherwise, add a target to the Podfile that embeds these frameworks to make this message go away (e.g. a test target).

I downloaded the project, run pod install, got this message. Should I ignore it ? or do what it says? Any suggestions ?

**Comment by mbalex99** [comment]
I cannot replicate this. 

Have you tried the latest version 
`pod 'NMessenger', '~> 1.0.79'`

**Comment by yarodevuci** [comment]
@mbalex99 
I don't. All i did is `pod 'NMessenger'`

**Comment by mbalex99** [comment]
Just making sure we are on the right version

* Are you using Swift 3?
* Are you using the latest pod. (try 1.0.79) and then make sure you run `pod repo update` before you rerun `pod install`

**Comment by yarodevuci** [comment]
@mbalex99 
Yes, I am using Swift 3 and latest XCode including latest version of Cocoa
Yes, I have the latest pod version as well.

**Comment by mbalex99** [comment]
I mean I guess we can ignore it for now. I am not seeing this warning on my end...
